.env.example

```diff
@@ -3,4 +3,5 @@ DISCORD_CLIENT_SECRET=Discord client secret here, gotten from the OAuth2 page on
 DISCORD_TOKEN=The Discord bot token, acquired from the bot page on your app.
 JWT_SECRET=The JWT secret, used to sign the session tokens. You can generate a secret by doing `require('crypto').randomBytes(64).toString('hex')`.
 DISCORD_CLIENT_ID=Discord client id here, gotten from the General Information page on your app on the Discord developer website.
-DISCORD_PUBLIC_KEY=Discord client secret here, gotten from the General Information page on your app on the Discord developer website.
\ No newline at end of file
+DISCORD_PUBLIC_KEY=Discord client secret here, gotten from the General Information page on your app on the Discord developer website.
+BASE_URL=The base url of the repldex wiki, eg https://v3.repldex.com
```

Comment:
This should mention that there should be no trailing slash at the end of the url
"url" and "repldex" should be capitalized, "e.g." instead of "eg", there should be a period at the end of the sentence.

Reply:
imma just mark this as resolved, even though it isnt